packages/opentelemetry-sdk-setup/package.json

```diff
@@ -38,6 +38,7 @@
     "@opentelemetry/core": "~1.24.0",
     "@opentelemetry/resources": "~1.24.0",
     "@opentelemetry/semantic-conventions": "~1.24.0",
+    "read-pkg-up": "^9.0.0",
```

Comment:
This was not listed as a dependency at all, even though it's used. I don't think that should be the case.